hydroshare/irods/ruleGenerateBagIt_HS.r

```diff
@@ -81,12 +81,16 @@ generateBagIt {
   msiFreeBuffer("stdout");
 
   ### - writes tagmanifest file to BAGITDATA/tagmanifest-md5.txt
-  msiDataObjChksum("*BAGITDATA" ++ "/bagit.txt", "forceChksum", *CHKSUM);
+  msiDataObjChksum("*BAGITDATA" ++ "/bagit.txt", "forceChksum=", *CHKSUM);
   writeString("stdout", *CHKSUM);
   writeLine("stdout", "    bagit.txt")
-  msiDataObjChksum("*BAGITDATA" ++ "/manifest-md5.txt", "forceChksum", *CHKSUM);
+  msiDataObjChksum("*BAGITDATA" ++ "/manifest-md5.txt", "forceChksum=", *CHKSUM);
   writeString("stdout", *CHKSUM);
   writeLine("stdout", "    manifest-md5.txt");
+  ### - write readme.txt file checksum to BAGITDATA/tagmanifest-md5.txt as readme.txt is included in all HydroShare resource bags
```

Comment:
Since this is an iRODS rule that was adapted from the rule in iRODS repo which uses three # for comments, I am doing the same to conform with their standard. This is not a python file, so not subject to pylint style check anyway, so I'll just keep it the way it is to conform with its counterpart iRODS rule style.